modules/involvementRequests.js

```diff
@@ -0,0 +1,52 @@
+const keyApi = 'zHktWsaRXYV3xlOoINt8';
+const giveLike = async (id) => {
+  const request = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${keyApi}/likes`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  return request.status;
+};
+
+const getLikes = async () => {
+  try {
+    const request = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${keyApi}/likes`);
+    const response = await request.json();
+    return response;
+  } catch (e) {
+    return [];
+  }
+};
+
+const getComments = async (id) => {
+  const request = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${keyApi}/comments?item_id=${id}`);
+  const response = await request.json();
+  return response;
+};
+
+// eslint-disable-next-line camelcase
```

Comment:
Please don't disable the linters. Consider fixing the linter issues instead of disabling the linters.